event/event.go

```diff
@@ -6,9 +6,8 @@ type InstanceComplete struct {
 	CantabularBlob string `avro:"cantabular_blob"`
 }
 
-// CommonOutputCreated provides an avro structure for an Output Created event
-type CommonOutputCreated struct {
-	FilterID   string `avro:"filter_output_id"`
+// CsvCreated provides an avro structure for a CSV Created event
+type CsvCreated struct {
```

Comment:
This name should probably be capitalised as in CSVCreated

Reply:
Also renaming the corresponding schema.